Enable long term statistics

harrymayr opened the pull request:
Of course, the other devices need to have also these changes

FireSon commented:
@harrymayr Thanks for the PR! Sorry I was a bit in a hurry 😄 but much appriciated! Updated all the other devices as well!

harrymayr commented:
Is there a reason, why you didn’t change batVolt and electricLevel on the other devices?

Von: FireSon ***@***.***>
Gesendet: Samstag, 19. April 2025 18:36
An: FireSon/Zendure-HA ***@***.***>
Cc: harrymayr ***@***.***>; Mention ***@***.***>
Betreff: Re: [FireSon/Zendure-HA] Enable long term statistics (PR #117)


@harrymayr<https://github.com/harrymayr> Thanks for the PR! Sorry I was a bit in a hurry 😄 but much appriciated! Updated all the other devices as well!

—
Reply to this email directly, view it on GitHub<https://github.com/FireSon/Zendure-HA/pull/117#issuecomment-2816775354>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AY3E6BMSDLPM32KGDR5737D22J3QVAVCNFSM6AAAAAB3OREJJ6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMJWG43TKMZVGQ>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
[https://avatars.githubusercontent.com/u/6523983?s=20&v=4]FireSon left a comment (FireSon/Zendure-HA#117)<https://github.com/FireSon/Zendure-HA/pull/117#issuecomment-2816775354>

@harrymayr<https://github.com/harrymayr> Thanks for the PR! Sorry I was a bit in a hurry 😄 but much appriciated! Updated all the other devices as well!

—
Reply to this email directly, view it on GitHub<https://github.com/FireSon/Zendure-HA/pull/117#issuecomment-2816775354>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AY3E6BMSDLPM32KGDR5737D22J3QVAVCNFSM6AAAAAB3OREJJ6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMJWG43TKMZVGQ>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
